### PR TITLE
Fix/fix minor issues

### PR DESCRIPTION
**Pull request by Vietprole**
# Fix the following:
- The Customer that just added should be on the first record in the list.
- Fix bug "No customers found" after searching.
- Add scrollbar for modal.

# Temporary deployment at 
- [https://javascript-training-internship.vercel.app/](https://javascript-training-internship.vercel.app/)

### CODE REVIEW

File: src/js/templates/dashboard.js [hunk 1]
```diff
@@ -210,10 +210,19 @@ function showNoCustomersFound() {
   tableBody.appendChild(noCustomersFound);
 }
 
+function deleteNoCustomersFound() {
```
Comment (hoangle-agilityio) [inline src/js/templates/dashboard.js:213]: should be `removeNoCustomersFound`

File: src/css/sections/modal.css [hunk 2]
```diff
@@ -11,6 +11,31 @@
   background-color: var(--white);
 }
 
+@media (max-height: 608px) {
```
Comment (hoangle-agilityio) [inline src/css/sections/modal.css:14]: what is it?

File: src/js/templates/dashboard.js (hunk 1 above)
Comment (Vietprole) [reply src/js/templates/dashboard.js:213]: Ok, I've renamed it.

File: src/css/sections/modal.css (hunk 2 above)
Comment (Vietprole) [reply src/css/sections/modal.css:14]: It is to style the modal when the screen size is smaller than the modal, allow users to scroll to view all parts of the modal.

**Pull request merged**